### PR TITLE
Update the charter to include solution exploration

### PR DESCRIPTION
**Pull request by rayankans**
As agreed upon during the CG meeting of TPAC 2022, the group will, in addition to exploring usages & challenges, begin looking into the solution space.

### CODE REVIEW

**Comment by rayankans** [comment]
Hey @QingAn, @dontcallmedom. Do you mind taking a look at the proposed charter for the next phase of the CG?

When it's ready we can kick off the charter amendment process / vote.

**Review by dontcallmedom — APPROVED** [review]

**Comment by dontcallmedom** [comment]
also paging @tidoust that has offered to help with this rechartering (since my own availability risks being erratic in the weeks to come)

File: charter.md [hunk 1]
```diff
@@ -2,30 +2,31 @@
...
 
-The group will not publish Specifications on topics other than those listed under Specifications above. See below for [how to modify the charter](#amendments-to-this-charter).
-
-
 ### Out of Scope
 
-Specific development of technical proposals to mitigate identified issues are out of scope of this charter; the group expects to [amend its charter](#amendments-to-this-charter) with a specific list of mitigations it would seek to address in a second phase of work, based on the analysis conducted as described above.
+The following things are out of scope:
+* Proposals that are not applicable to WebViews as defined in this charter (e.g. proposals on browser related improvements),
+* Technical Specifications
```
Comment (tidoust) [inline charter.md]: The nuance between "technical proposals" (in scope) and "technical specifications" (out of scope) would perhaps benefit from being clarified. I suggest:

```suggestion
* Normative specifications. Actual incubation of the explainers developed by the group is not in scope of this charter.
```

**Comment by rayankans** [comment]
Thanks @tidoust, this is much clearer now.

**Review by QingAn — APPROVED** [review]

**Comment by rayankans** [comment]
Sorry folks, accidentally merged this and had to revert. Reland is in #10. @tidoust, after your approval (% any additional comments) I'll kick off the vote.

**Pull request merged**